pygmt/tests/test_info.py

```diff
@@ -41,6 +41,18 @@ def test_info_dataframe():
     assert output == expected_output
 
 
+def test_info_numpy_array_time_column():
+    """
+    Make sure info works on a numpy.ndarray input with a datetime type.
+    """
+    table = pd.date_range(start="2020-01-01", periods=5).to_numpy()
+    output = info(table=table, f="0T")
```

Comment:
The `f="0T"` argument can be removed once upstream GMT fixes https://github.com/GenericMappingTools/gmt/issues/4241.